docs/reference/linearmotor.md

```diff
@@ -26,3 +26,5 @@ It is expressed in *Newton* (N).
 The `wb_motor_set_available_force` function can be used to change the *available force* at run-time.
 The value of `maxForce` should always be zero or positive (the default value is 10 N).
 A small `maxForce` value may result in a motor being unable to move to the target position because of its weight or other external forces.
+
+    In [*kinematics* mode](physics.md#how-to-use-physics-nodes), `maxForce` is used to clamp the joint acceleration in case `Motor.acceleration` is not defined.
```

Comment:
I'm not sure this sentence describes precisely what it happens.

Is the clamp done only if the acceleration is not defined?
from the code it seems that the clamp is applied if the acceleration is defined but it is bigger than the force/torque.
And then if the acceleration is not defined, it seems that the acceleration is not clamped but set to `maxForce`.
https://github.com/cyberbotics/webots/blob/faa4a29d04c5c21863c23d05d266fe56626671fb/src/webots/nodes/WbMotor.cpp#L299-L300

Reply:
Very true, I will rephrase this.